depobs/website/views.py

```diff
@@ -16,29 +17,29 @@
 app.register_blueprint(tasks_api)
 
 
-@app.route('/package/<pkgname>/<version>')
-def show_package_by_name_and_version(pkgname, version):
-    package_report = models.get_package_report(package = pkgname, version = version)
-    if None != package_report:
-        mimetype = "application/json"
-        return Response(json.dumps(package_report.json_with_dependencies()), headers=STANDARD_HEADERS, mimetype=mimetype)
-    else:
-        #TODO: we probably want to return data to tell the user that a report is being generated
-        abort(404)
+@app.errorhandler(BadRequest)
+def handle_bad_request(e):
+    return dict(description=e.description), 400
+
+
+@app.route('/packages', methods=["GET"])
```

Comment:
Is it really packages? We're just checking one package and version arent we?

Reply:
I thought the restful way was usually `<plural of collection name>/<collection item id>` or `<plural of collection name>?<filters on that collection>`

Reply:
changed it back per conversation in chat